MAUI/PDF-Viewer/Toolbar.md

```diff
@@ -934,7 +938,7 @@ To hide a toolbar item by its index, you can directly access the item in the too
 int indexToHide = 2; // Replace with the actual index of the item you want to hide.
 
 var toolbar = pdfViewer.Toolbars?.GetByName("PrimaryToolbar");
-if (toolbar != null && indexToHide >= 0 && indexToHide < toolbar.Items.Count)
+if (toolbar != null && indexToHide >= 0 && indexToHide < toolbar?.Items?.Count)
```

Comment:
toolbar  was null checked before, why we need to null check again?
<img width="280" alt="image" src="https://github.com/user-attachments/assets/5eee4de0-67c2-4bfc-ab5d-a947c77152e1">